src/browser/api/system.js

```diff
@@ -163,15 +163,14 @@ export const System = {
         };
 
         electronApp.vlog(1, `clearCache ${JSON.stringify(storages)}`);
-        clearCacheInvoked(true);
 
-        defaultSession.clearCache(() => {
-            defaultSession.clearStorageData(cacheOptions, () => {
-                resolve();
+        grantAccess(async () => {
```

Comment:
Do we need at `catch` or a `finally` on the `clearCache` call or will that throw and resolve (nack)? Also, do we need that await there?

Reply:
I will add try-catch block.  
await is needed here.

Reply:
Perfect, lgtm. Once the tests clear I'll merge